### DISCUSSION TITLE
Crazyflie Bolt PWM Problem

### DISCUSSION THREAD

**Discussion by TigeyJewellAlibhai** [bitcraze/discussions#629] [Q&A]
Hello! I am working on integrating a bolt into two different quadcopters. The first is a holybro kit that works great after some wire harnessing. The bolt interfaces nicely with the ESCs. 

I am also working on a large quad that has APD F3X escs, and things are not going so smoothly. A servo tester has proved that the escs work great, but when plugged into the bolt, they dont even detect a pwm input (despite this being the same bolt that I put on the holybro, that I know works). When I run a pwm monitor to check what the bolt is outputting, the value range is ~125 to ~240, whereas a normal servo tester outputs ~800 to ~2000. Is this normal? Is this something the esc should be able to read? I have triple checked all wiring.

Thank you!

**Comment by krichardsson** [comment]
Hi!
Our Bolt expert @tobbeanton is out travelling and will not be in the office until next week. I'm not sure he is checking the discussions while away, so unfortunately it might take a while before you get an answer.

**Reply by knmcguire** [reply]
if you haven't heard back in a week, please send us an reminder ;) These things might get lost and just sending us a message here will automatically put it on our attention list.
